src/core/util/util.js

```diff
@@ -874,6 +874,16 @@ function UtilFactory($document, $timeout, $compile, $rootScope, $$mdAnimate, $in
       return Array.prototype.map.call(element.childNodes, function (child) {
         return serializer.serializeToString(child);
       }).join('');
+    },
+
+    /**
+     * Function to get outerHTML of SVG and Symbol elements in IE11
+     * @param {Element} element
+     * @returns {string} the outerHTML of the element passed in
+     */
+    getOuterHTML: function(element) {
```

Comment:
Why not just use the `outerHTML` property?
https://developer.mozilla.org/en-US/docs/Web/API/Element/outerHTML

Reply:
It's not supported by IE11 for SVG elements. It returns `undefined`.

Reply:
It's not mentioned in https://caniuse.com/#feat=xml-serializer, but I've left feedback that it should be added to the Known Issues.

Reply:
Reference: https://stackoverflow.com/questions/29888050/outerhtml-is-not-working-in-ie.
Also manually tested.

Reply:
Got it. I would just add an line comment here that adds all of that context to the code. The function description should just be `Gets the outer HTML content of the given HTMLElement`